src/components/Place/PlaceComponents/PlaceDetails/PlaceDescription.js

```diff
@@ -4,11 +4,11 @@ const CURRENCY_NAME = "zł"
 
 const PlaceDescription = ({address, city, description, price, rate}) => {
 
-return ( <div className="Card__Container__description">
-    <p className="Card__Container__description__item"><i class="fas fa-star"></i> {rate}</p>
-    <p className="Card__Container__description__item">{description}</p>
-    <p className="Card__Container__description__item">- {address}, {city}</p>
-    <p className="Card__Container__description__item">- {price} {CURRENCY_NAME}</p>
+return ( <div className="Card__Container__Description">
```

Comment:
According to BEM methodology, you shouldn't nest Blocks. So, you should have something like: Card, then Card_Item, Card_Description, without all the things in between.

Reply:
@weronikaolejniczak @lukaszpetra @MaciejPaszkowski @pawelBareja @Arturk89 

Since someone started to use BEM methodology in project, lets start to use BEM and we need to create new issue to refactor all classes and implement BEM.